bowtie/_report.py

```diff
@@ -52,6 +54,32 @@ class MissingFooter(InvalidReport):
     """
 
 
+class CaseMismatchInSameSuiteReports(Exception):
+    """
+    Two reports for the same dialect suite had some Case mismatch in them.
+    """
+
+
+class SeqResultMismatchInSameSuiteReports(Exception):
+    """
+    Two reports for the same dialect suite had some SeqResult mismatch.
+
+    **Condition to that it was the same ConnectableId (i.e. same implementation
+    and its same version) in both the reports, yet there was some mismatch in
+    its SeqResults.**
+    """
+
+
+class ImplementationInfoMismatchInSameSuiteReports(Exception):
+    """
+    Two reports for the same dialect suite had an ImplementationInfo mismatch.
```

Comment:
And here -- what makes this an error? Maybe an implementation added a link or something, does that really need to be an error?

In general I think just seeing these makes me think maybe you can have a separate PR (first?) which adds whatever additional validation motivated you to add these?